src/app/static/src/tests/testHelpers.ts

```diff
@@ -96,6 +97,11 @@ export const expectTranslated = (element: Wrapper<any>,
                                  attribute?: string) =>
     expectTranslatedWithStoreType<DataExplorationState>(element, englishText, frenchText, portugueseText, store, attribute);
 
+export const expectHasTranslationKey = (element: Wrapper<any>, translationKey: keyof Translations) => {
```

Comment:
Follwing on from Rob's suggestion that testing every translation string is OTT, how about this as an alternative, to check that an element has the expected translation key?

Could be accompanied by a test of the locales strings to check that each key has different value in each language.

Reply:
I think this is a better approach!

Reply:
I think testing every string isn't necessary. The current test approach means every time a string changes you have to update several tests which seems more pain than it is worth. We have tests that the directive works so we should rely on that, I don't know what testing every string in each language gives us besides loads of churn when we want to update a string. We've seen encoding issues on production even when we're testing every translation so I'm not sure they even guard us against that. Maybe we can try this and if we see missing translations in the app then we can revert back to the old approach of testing every string?